.github/workflows/versioning.yaml

```diff
@@ -32,4 +32,7 @@ jobs:
         yarn config set version-commit-hooks false
         yarn version --new-version `node ./tools/getNextVersion.js`
         git push --tags --no-verify &&
+        yarn create:changelog
```

Comment:
i think, if move `yarn create:changelog` before `yarn version` we can create change log and bump version with one commit.

Reply:
Yeah, that's a good point